aten/src/ATen/native/SoftMax.cpp

```diff
@@ -34,6 +35,27 @@ TORCH_META_FUNC(_log_softmax) (
       dim_ >= 0 && dim_ < input_.dim(),
       "dim must be non-negative and less than input dimensions");
 }
+
+TORCH_META_FUNC(_log_softmax_backward_data)
+(const Tensor& grad,
+ const Tensor& output,
+ int64_t dim,
+ const Tensor& input){
+  auto grad_ = grad.contiguous();
```

Comment:
Call to contiguous() in meta function is a no no. That's compute; do it in impl func only.